### ISSUE TITLE
Change cron to download postgrey_whitelist_clients to hash match

### ISSUE THREAD

**Issue by jvolkenant** [mail-in-a-box/mailinabox#2072]
We should change this code block to just download the postgrey_whitelist_clients file if it doesn't match the destination hash.  The postgrey_whitelist_clients file hasn't been updated since 2019 (https://github.com/schweikert/postgrey/blob/master/postgrey_whitelist_clients)

I think the likelihood of it being updated in the future is slim, and if it is, we can update the hash and the next update would pull down a new version of the file.

https://github.com/mail-in-a-box/mailinabox/blob/50a5cb90bc63d3f492641bf39ed943886f9c4eab/setup/mail-postfix.sh#L246

**Comment by myfirstnameispaul** [comment]
I think the URL in `mail-postfix.sh` is the correct one to use (not their GitHub).

**Comment by jvolkenant** [comment]
True.  I posted the github link to show it's update history.  The file from the project homepage and github are the same files.

```
root@pdxutil:~# curl -s https://postgrey.schweikert.ch/pub/postgrey_whitelist_clients | sha1sum
f2f27e75249ed30ebf93363e4fd63c6208c23266  -
root@pdxutil:~# curl -s https://raw.githubusercontent.com/schweikert/postgrey/master/postgrey_whitelist_clients | sha1sum
f2f27e75249ed30ebf93363e4fd63c6208c23266  -

```
